fern/api/definition/device.yml

```diff
@@ -23,7 +23,27 @@ services:
             user_id:
               type: ids.UserId
               docs: your user identifier
-          request: Device
+          request:
+            name: AddDeviceRequest
+            body:
+              properties:
+                id: optional<string>
+                platform: optional<Platform>
+                onesignal_player_id: optional<string>
+                xiaomi_token: optional<string>
+                oppo_token: optional<string>
+                vivo_token: optional<string>
+                huaweiToken: optional<string>
+                fcm_token:
+                  type: optional<string>
+                  docs: firebase device token
+                raven_id:
+                  type: optional<string>
+                  docs: user id affiliated with device
+                device_sid: optional<ids.DeviceId>
+                notifications_disabled: optional<boolean>
+                created_at: optional<long>
+                updated_at: optional<long>
```

Comment:
i think we should make this
```yaml
request: 
  body: 
    type: Device
```

Reply:
you can also just do `request: Device` for shorthand